client/src/components/related_products/row1/RelatedCard.jsx

```diff
@@ -59,7 +59,7 @@ function RelatedCard({ relatedID, changeProduct }) {
   }, [relatedID]);
 
   return (
-    <div className="card" role="button" tabIndex="0" onKeyDown={() => {}} onClick={() => changeProduct(relatedID)}>
+    <div className="cardContainer" role="button" tabIndex="0" onKeyDown={() => {}} onClick={() => changeProduct(relatedID)}>
```

Comment:
being explicit with the class name here is great and will make this easier for other programmers to read and update in the future. nice work!